docs/api.md

```diff
@@ -725,6 +735,40 @@ Determines whether or not boss bar creates fog
 
 Determines what color the boss bar color is, one of `pink`, `blue`, `red`, `green`, `yellow`, `purple`, `white`
 
+### mineflayer.Particle
+
+#### Particle.id
+
+Particle ID, as defined in the [protocol](https://wiki.vg/Protocol#Particle)
+
+#### Particle.name
+
+Particle Name, as defined in the [protocol](https://wiki.vg/Protocol#Particle)
+
+#### Particle.forceRender
+
+Determines whether or not to force the rendering of a particle despite client particle settings and increases maximum view distance from 256 to 65536
```

Comment:
`forceRender` is not a great name, because it has an extended but not unlimited rage

Reply:
Thought that would be better suited given it was the primary purpose and the tab complete called the option for it `force`, changed to `longDistanceRender` as suggested